File: Java/Array/program22.java
//Accept the N*N elements from user and check whether the given element is present or not
import java.util.*;

class ArrayX
{
    public int iRow;
    public int iCol;
    public int Arr[][];

    ArrayX(int iValue1, int iValue2)
    {
        iRow = iValue1;
        iCol = iValue2;

        Arr = new int[iRow][iCol];
    }

    public void Accept()
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the elements of array");

        for(int i = 0; i < iRow; i++)
        {
            for(int j = 0; j < iCol; j++)
            {
                Arr[i][j] = sobj.nextInt();
            }
        }
        
    }

    public boolean Search(int iValue)
    {
        for(int i = 0; i < Arr.length; i++)
        {
            for(int j = 0; j <Arr[i].length; j++)
            {
                if(Arr[i][j] == iValue)
                {
                    return true;
                }
            }
        }
        return false;
    }
}

class program22
{
    public static void main(String arg[])
    {
        int iValue1 = 0;
        int iValue2 = 0;
        int iNo = 0;
        boolean bRet;

        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the number of rows");
        iValue1 = sobj.nextInt();

        System.out.println("Enter the number of columns");
        iValue2 = sobj.nextInt();

        ArrayX obj = new ArrayX(iValue1,iValue2);

        obj.Accept();

        System.out.println("Enter the element to be search");
        iNo = sobj.nextInt();

        bRet = obj.Search(iNo);
        if(bRet == true)
        {
            System.out.println("Element is there in array");
        }
        else
        {
            System.out.println("Element is not there in array");
        }
        
    }
}

File: C/Array/program22.c
//Accept the N*N numbers from user and display that numbers

#include<stdio.h>

int main()
{
    int iRows = 0, iCols = 0;
    int i = 0, j = 0;

    printf("Enter the number of rows\n");
    scanf("%d",&iRows);

    printf("Enter the number of columns\n");
    scanf("%d",&iCols);

    int Arr[iRows][iCols];

    printf("Enter the values of array\n");

    for(i = 0; i < iRows; i++)
    {
        for(j = 0; j < iCols; j++)
        {
            scanf("%d",&Arr[i][j]);
        }
    }

    for(i = 0; i < iRows; i++)
    {
        for(j = 0; j < iCols; j++)
        {
            printf("%d\t",Arr[i][j]);
        }
        printf("\n");
    }

    return 0;
}